content/2_Bronze/Intro_Sets.mdx

```diff
@@ -73,7 +73,7 @@ already present, `remove`, which deletes an element if it exists, and
 `contains`, which checks whether the set contains that element.
```

Comment:
Spacing on line 88, before the opening parenthesis, and then after the closing parenthesis